data/systemd/meson.build

```diff
@@ -0,0 +1,11 @@
+configure_file(
+  input: 'org.gnome.GPaste.Ui.systemd.in',
```

Comment:
Any way to loop over `*.systemd.in` instead of listing manually the files?

Reply:
Again, simplicity > being smart about it IMO